src/language/worker.ts

```diff
@@ -67,6 +78,17 @@ const execute = async (
       },
     });
 
+    // In order to support local require direclty from the file where code lives, we can not wrap the
+    // whole code in a function for two reasons:
+    // 1. We need to return the response and can not simply add return. And
+    // 2. We can not use eval to evaluate the *codeToEvaluate* as mongosh async-rewriter can’t see into the eval.
+    // We are also not direclty concatenating the require with the code either due to "use strict"
+    if (filePath) {
+      await runtime.evaluate(`(function () {
+        require = require('module').createRequire('${filePath}');
```

Comment:
```suggestion
        globalThis.require = require('module').createRequire(${JSON.stringify(filePath)});
```

- Using `JSON.stringify()` is the easiest way to make sure that strings are inserted literally when programmatically creating JS code, it takes care of all necessary escaping – without it, any file path containing a `\` or `'` would break, and the former is obviously pretty common on Windows 🙂 
- We're assigning to the global object here anyway, but that might not be obvious to readers, so let's make it obvious 🙂